winterfell/controllers/claimController.js

```diff
@@ -94,15 +100,55 @@ module.exports = function (app) {
     }
   }
 
-  app.post('/claim/:extClaimId/submit', function(req, res) {
+  app.post('/claim/:extClaimId/submit',  auth.authenticatedOr404, function(req, res) {
     console.log('[submitClaim] request received ' + JSON.stringify(req.body));
+    var that = this;
+    var currentUser = null;
+    var promise = User.findById(req.session.userId);
 
-    handleClaimStateChange(req.params.extClaimId,
-      Claim.State.SUBMITTED,
-      claimUpdateCallbackFactory(res));
+    promise = promise.then(function(user) {
+      currentUser = user;
```

Comment:
Yeah I need to go back and standardize handling of that across all the controllers, we haven't been too strict about it so far.